canonical-web-and-design/ubuntu.com
# Update text and design of /appliance/community page

## Done

- [List of work items including drive-bys.]

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/community
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Cpmpare to the [copy doc](https://docs.google.com/document/d/1DpU0AzAJ4gBa_au4gELBUD3vLz8l-FnAwFdERBHlvwc/edit#) and the [zeplin design](https://app.zeplin.io/project/5eb3d16a91c4117789940141/screen/5ee21409486d3d9a9c4194bf)
